src/ES.SFTP.Host/Configuration/Elements/UserDefinition.cs

```diff
@@ -7,6 +7,7 @@ public class UserDefinition
         public string Username { get; set; }
         public string Password { get; set; }
         public bool PasswordIsEncrypted { get; set; }
+        public string Host { get; set; }

```

Comment:
Can this be a list of hosts? Also can we rename to `AllowedHost` (possible plural if a list can be used)?

Reply:
Changed to a list, this can be a mix of CIDR, IP addresses (See PATTERNS in ssh_config documentation https://manpages.debian.org/stretch/openssh-client/ssh_config.5.en.html#PATTERNS) I believe that using DNS names won't work with option `UseDNS no`

Reply:
@antoinedeschenes If I remember correctly, UseDNS has a major performance hit. I think IPs and CIDRs are more than enough. Great contribution!